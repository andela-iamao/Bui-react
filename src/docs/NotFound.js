import React from 'react';


export default () => {
    return (
        <div id="bui-react-docs-component-not-found">
            <h3>404!</h3>
        </div>
    );
}
