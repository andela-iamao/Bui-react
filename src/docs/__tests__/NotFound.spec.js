import React from 'react';
import renderer from 'react-test-renderer';

import { shallow } from 'enzyme';
import NotFound from '../NotFound';


describe('DOCS: NotFound', () => {

    describe('Snapshot Test', () => {

        test('renders a 404 message on page', () => {
            const tree = renderer.create(<NotFound />).toJSON();
            expect(tree).toMatchSnapshot();
        });

    });

    describe('Unit Test', () => {

        test('renders a 404 message on page', () => {
            const wrapper = shallow(<NotFound />)
            expect(wrapper.find('#bui-react-docs-component-not-found h3')).toHaveLength(1);
        });

    });
});
