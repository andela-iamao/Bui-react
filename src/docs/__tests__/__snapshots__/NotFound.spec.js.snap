// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DOCS: NotFound Snapshot Test renders a 404 message on page 1`] = `
<div
  id="bui-react-docs-component-not-found"
>
  <h3>
    404!
  </h3>
</div>
`;
